src/SignalR/clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -499,11 +500,12 @@ describe("hubConnection", () => {
 
                 const complexObject = {
                     ByteArray: protocol.name === "json"
-                        ? new Array(0x1, 0x2, 0x3)
+                        ? "AQID"
                         : new Uint8Array([0x1, 0x2, 0x3]),
                     DateTime: protocol.name === "json"
                         ? "2000-01-01T00:00:00Z"
                         : new Date(Date.UTC(2000, 0, 1)),
+                    GUID: "00010203-0405-0607-0706-050403020100",
```

Comment:
These test changes aren't random. Re-adding coverage now that features exist: https://github.com/aspnet/AspNetCore/commit/c84e37f30def9ff0b2d12e877e3de6c283be6145#diff-1c30bdd79e21ac4ce12ab9663aa438e0L508